src/cascadia/TerminalSettingsModel/AppearanceConfig.cpp

```diff
@@ -50,6 +54,19 @@ Json::Value AppearanceConfig::ToJson() const
     JsonUtils::SetValueForKey(json, SelectionBackgroundKey, _SelectionBackground);
     JsonUtils::SetValueForKey(json, CursorColorKey, _CursorColor);
     JsonUtils::SetValueForKey(json, OpacityKey, _Opacity, JsonUtils::OptionalConverter<double, IntAsFloatPercentConversionTrait>{});
+    if (HasDarkColorSchemeName() && HasLightColorSchemeName())

```

Comment:
Hmm... If the JSON has...
```js
"colorScheme": {
   "dark": "foo"
// omit "light"
}
```

And you hit the save button in SUI, you lose the entire "colorScheme" object entirely.

I think we need this statement to be `||` and we need to check if each one exists before writing it.

Normally I wouldn't care, but this is data loss so I think it's kinda important.